### DISCUSSION TITLE
Unity v6 - UnityEditor.Rendering.HighDefinition.OpaqueCullMode not found

### DISCUSSION THREAD

**Discussion by 442FX** [visualdesigncafe/nature-renderer#86] [General]
Does anyone have insight into this issue, or is it just a me thing?

Appreciate any response ...from anyone ...seriously.

https://github.com/visualdesigncafe/nature-renderer/issues/76

**Comment by frostisenjostein** [comment]
Also have this issue. No idea what to do with it.

**Comment by maxplooi** [comment]
It's a warning from one of the shaders in the HDRP demo, it is safe to ignore and does not cause any issues. One of the properties in the shaders uses an outdated enum for the inspector UI, so the enum dropdown will not show when selecting the material. It has absolutely no impact on Nature Renderer or any rendering.
